package/tests/test_client.py

```diff
@@ -1,13 +1,63 @@
 import unittest
+import os
 from aistore.client.api import Client
+from aistore.client.msg import Bck
+from aistore.client.const import PROVIDER_AIS
 
 
-class TestStringMethods(unittest.TestCase):
+class TestBasicPutGet(unittest.TestCase):
     def test_list_objects(self):
         client = Client('http://localhost:8080')
-        objects = client.list_objects('test')
-        self.assertEqual(objects, ['a', 'b', 'c'])
 
+        res = client.list_buckets(provider=PROVIDER_AIS)
+        print("Bucket list:")
+        for bck in res:
+            print("    ", bck["name"])
+
+        print("Creating bucket 'test'")
+        bck = Bck('test', PROVIDER_AIS)
+        res = client.create_bucket(bck)
+        if res.status_code != 200:
+            print("Bucket cretion failed:", res)
+            return
+
+        tmpfile = "/tmp/py-sdk-test"
+        orig_cont = "test string"
+        with open(tmpfile, "w") as fdata:
+            fdata.write(orig_cont)
+
+        print("Creating an object 'obj1'")
+        res = client.put_object(bck, "obj1", tmpfile)
+        os.remove(tmpfile)
+        if res.status_code != 200:
```

Comment:
I believe we should be using `self.asserEqual` in all our tests instead of just returning. If I am not wrong, this test will never fail.  

Also, I feel it makes sense for us to separate integration and unit tests. We can look into how we structure our test suite outside this MR.

Reply:
Removed all `print`s and use test library asserts